Cython/Compiler/ParseTreeTransforms.py

```diff
@@ -889,6 +890,12 @@ def _check_valid_cython_module(self, pos, module_name):
             if module_name.startswith("cython." + wrong):
                 extra = "Did you mean 'cython.%s' ?" % correct
                 break
+        if not extra:
+            from .. import Shadow
+            if (submodule in Shadow.__dict__
+                    or submodule in Options.directive_types):
+                extra = ("Instead, 'cimport cython' and use 'cython.%s'." %
+                    submodule)
```

Comment:
Should we use propose a plain `import` instead of a `cimport` here, in order to avoid confusion in pure Python code?
```suggestion
                extra = "Instead, use 'import cython' and then 'cython.%s'." % submodule
```

Reply:
I'm fine with this. When I originally wrote this I tried to include/omit the `c` based on if we're in a py or pyx file but this information didn't seem available at that point.

So lets just change it to `import`